isabelle/Core/SemanticsTypes.thy

```diff
@@ -367,29 +258,79 @@ and Contract = Close
              | Let ValueId Value Contract
              | Assert Observation Contract
 
-type_synonym Accounts = "((AccountId \<times> Token) \<times> Money) list"
+text \<open>@{term Close} is the simplest contract, when we evaluate it, the execution is completed and we
+generate \<^term>\<open>Payment\<close>s \secref{TODO} for the assets in the internal accounts to their default owners
```

Comment:
Missing cross-reference.

Reply:
yes, this depends on wether we move `Payment` here, or if we split types into `DSL` and `Execution`